app.js

```diff
@@ -221,22 +219,22 @@ function createServer(app) {
 
 }
 
-
 /**
  * The service's entry point. It takes over the configuration
  * options and the logger- and metrics-reporting objects from
  * service-runner and starts an HTTP server, attaching the application
  * object to it.
+ * @param {Object} options the options to initialise the app with
+ * @return {bluebird} HTTP server
  */
-module.exports = function(options) {
+module.exports = function (options) {
```

Comment:
Nit: you could use the `() => {}` notation since you are changing the line.

Reply:
Done